House pricey.js

```diff
@@ -0,0 +1,16 @@
+let petersHouselengthInMeter = 8 ;
+let petersHousedepthInMeter = 10;
+let petesrHousewidthInMeyer = 10;
+let volumeOfPetersHouseInMeter = petersHouselengthInMeter*petersHousedepthInMeter*petesrHousewidthInMeyer;
+ let petersHouseGardenSizeInM2 = 100;
+ let house1cost = 2500000;
+ let houseprice = volumeOfPetersHouseInMeter * 2.5 * 1000 + petersHouseGardenSizeInM2 * 300;
+ console.log("Peter paying $" +  (house1cost-houseprice) + " lower price");
```

Comment:
It's better to utilize a `if else` statement checking if the `house1Cost` is > or = to the `housePrice` and depending on the result, to log one or another message.
Remember to write variable names in [camelCase](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#camel-case) for better readability.